src/components/answerComponent.js

```diff
@@ -2,10 +2,25 @@
  * Create an Answer element
  * @returns {Element}
  */
-export const createAnswerComponent = (key, answerText) => {
+import { setLocalStorage } from '../utils/setLocalStorage';
+
+export const createAnswerComponent = (
+  key,
+  answerText,
+  currentQuestion,
+  quizData
+) => {
   const element = document.createElement('li');
   element.innerHTML = String.raw`
-    ${key}: ${answerText};
+    <label for="${key}">
+      <input type="radio" name="answerItem" value=${key} id="${key}" />
+    ${answerText}
+    </label>
   `;
+  element.addEventListener('click', (e) => {
+    e.stopPropagation();
+    e.preventDefault();
+    setLocalStorage(currentQuestion, quizData, key);
```

Comment:
We agreed that we will keep the quizData outside of the context of `answer`.

Reply:
Instead of passing quizData, currentQuestion, you can pass a function called `onSelect`

and call it with `onSelect(key)`

From `questionPage.js`

```
createAnswerComponent(key, answerText, (selectedKey) => {
   setLocalStorage(currentQuestion, key);
})
```